katsdpcal/test/test_control.py

```diff
@@ -989,22 +989,29 @@ async def test_reference_pointing_capture(self):
                     ts = ts + (self.dump_period/8)
 
         # Creating antenna delays and gains
-        K = rs.uniform(-50e-12, 50e-12, (2, self.n_antennas))
-        G = (rs.uniform(2.0, 4.0, (2, self.n_antennas))
-             + 1j * rs.uniform(-0.1, 0.1, (2, self.n_antennas)))
-
+        K_nan = rs.uniform(-50e-12, 50e-12, (2, self.n_antennas))
+        G_nan = (rs.uniform(2.0, 4.0, (2, self.n_antennas)) +
+                 1j * rs.uniform(-0.1, 0.1, (2, self.n_antennas)))
```

Comment:
The current thinking is to have the binary operator (+) at the start of the second line when you break up a line:
```suggestion
             + 1j * rs.uniform(-0.1, 0.1, (2, self.n_antennas)))
```